cloud-annotations/src/main/java/cloud/commandframework/annotations/Flag.java

```diff
@@ -59,6 +63,22 @@
      */
     String parserName() default "";
 
+    /**
+     * Name of the suggestions provider to use. If the string is left empty, the default
+     * provider for the argument parser will be used. Otherwise,
+     * the {@link cloud.commandframework.arguments.parser.ParserRegistry} instance in the
```

Comment:
Import the links. This is supposed to be easily readable, even outside the javadoc page.